tools/sigmut/sigmut.xml

```diff
@@ -0,0 +1,495 @@
+				</param>
+			</when>
+
+			<when value="get_sigmut">
+				<conditional name="vcfile_input">
+					<param name="vcfile" type="select" label="VC file" help="Select the format of your input data">
+						<option value="maf">Mutation Annotation Format</option>
+						<option value="icgc">Tab-separated file</option>
+						<option value="vcf">Variant Call Format</option>
+					</param>
+					<when value='maf'>
+						<param name="maf_file" type="data" format="maf" label="select VC file" help="Select the input file in MAF format." />
+					</when>
+					<when value='icgc'>
+						<param name="icgc_file" type="data" format="txt" label="select VC file" help="Select the input file in ICGC format." />
+					</when>
+					<when value='vcf'>
+						<param name="vcf_file" type="data" format="vcf" label="select VC file" help="Select the input file in VCF format." />
+					</when>
+				</conditional>
+
+				<param name="refgendat" type="select" label="Reference genome to be analyzed" help="Use the following reference genome:">
+					<option value="GRCh37">GRCh37.p13 [GRCh37] </option>
+					<option value="GRCh38">GRCh38.p12 [GRCh38] </option>
+					<option value="mm9">GRCm37 [mm9] </option>
+					<option value="mm10">GRCm38.p6 [mm10] </option>
+				</param>
+
+				<conditional name="bed_input">
+					<param name="bedfile" type="select" label="BED file" help="Input a BED file">
+						<option value="yes">Yes</option>
+						<option value="no" selected="true">No</option>
+					</param>
+					<when value='yes'>
+						<param name="bed_file" format="bed" type="data" label="Use a BED file containing the set of regions" help="Provide a BED file"/>
+					</when>
+					<when value='no'>
+					</when>
+				</conditional>
+				<param name="plot" type="boolean" truevalue="true" label="Produce plot results?" checked="False" help="Show plots"/>
+				<param name="exome" type="boolean" label="Use only the exome?" checked="False" help="Use exome"/>
+				<param name="chrom_based" type="boolean" label="Create the matrices on a per chromosome basis?" checked="False" help="Show snvs"/>
+				<param name="tsb_stat" type="boolean" truevalue="true" label="Performs a transcriptional strand bias test?" checked="False" help="Show snvs"/>
+				<param name="seqInfo" type="boolean" truevalue="true" label="Export sequence information?" checked="False" help="Show sequence information"/>
+				<param name="gs" type="boolean" label="Performs gene strand bias test?" checked="False" help="Show snvs"/>
+			</when>
+		</conditional>
+	</inputs>
+
+	<outputs>
+		<data format="txt" name="logref" label="Log file: Install a Reference Genome">
+			<filter>set_analysis['choices'] == 'install_genome'</filter>
+		</data>
+
+                <data format="txt" name="logsmt" label="Log file: Calculate Mutational Signatures">
+                        <filter>set_analysis['choices'] == 'get_sigmut'</filter>
+                </data>
+
+		<data format="pdf" name="SBS6" label="--> Plot SBS 6 Sig. Mut.">
```

Comment:
why these weird names ? I there a biological or genomic or statistic implicit sense ?

Reply:
SBS : Single Base substitutions. The number is related to the amount of possible mutations.
The most basic classification catalogues SBSs into six distinct categories, including: C:G > A:T, C:G > G:C, C:G > T:A, T:A > A:T, T:A > C:G, and T:A > G:C. In addition, the SBS-6 classification can be further expanded by taking into account the base-pairs immediately adjacent 5′ and 3′ to the somatic mutation.